Source/Core/Section.swift

```diff
@@ -476,8 +476,11 @@ open class MultivaluedSection: Section {
     func initialize() {
         let addRow = addButtonProvider(self)
         addRow.onCellSelection { cell, row in
-            guard let tableView = cell.formViewController()?.tableView, let indexPath = row.indexPath else { return }
-            cell.formViewController()?.tableView(tableView, commit: .insert, forRowAt: indexPath)
+            if !row.isDisabled
```

Comment:
Can you leave this like `if !row.isDisabled {`, with the `{` in the same line, please?

Reply:
Done